tests/legacy-cli/e2e/tests/misc/supported-angular.ts

```diff
@@ -0,0 +1,54 @@
+import { SemVer } from 'semver';
```

Comment:
I am surprised that so much code is added in this PR. Didn't we do something similar in earlier versions?

Reply:
It was hard coded to check for angular 5.0+.
This file is on my list to refactor after the RC.  About half of it is either unused or unneeded.